docs/guides/routing.rst

```diff
@@ -18,6 +18,7 @@ available via ``get_input()``)
 The site URL (home page) is a special case that produces an empty string identifier and
 an empty segments array.
 
+.. warning:: URL identifier/segments are **not** filtered, and should be considered dangerous user input.
```

Comment:
I never thought of it but I don't see downsides. The chances of expected paths containing markup characters seems really low, and if it breaks something it probably should be broken for safety. Q is whether to run the whole path through htmlawed or each segment.